.github/workflows/test-ui-e2e-only-desktop.yml

```diff
@@ -30,16 +30,21 @@ on:
         description: Enable transaction broadcast
         type: boolean
         default: false
-
-      report_path:
-        description: Path where you want to store this execution's results
-        required: false
-        default: "allure-results-linux"
       slack_notif:
         description: "Send notification to Slack?"
         required: false
         type: boolean
         default: false
+      export_to_xray:
+        description: Send tests results to Xray
+        required: true
```

Comment:
I don't think export_to_xray and test_plan should be required to execute the tests

Reply:
➕

Reply:
done :)

Reply:
`export_to_xray` input shouldn't be required

Reply:
I agree with @bharamboure-ledger

Reply:
I just change it to false